test/run-jest.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -x
+set -e
+set -u
+set -o pipefail
+
+# For storing PIDs of subshells
+PIDS=(0 0)
+
+yarn 'jest' '--config=./jest.config.js' "$@" & PIDS[0]=$!
+yarn 'jest' '--config=./development/jest.config.js' "$@" & PIDS[1]=$!
+
+# This ensures that this shell exists with a non-zero code if any of the
+# subshells we waited for did
+for pid in ${PIDS[*]}; do
```

Comment:
This is giving ShellCheck some difficulty due to word splitting. Here you're relying on word-splitting for the array elements to be correctly distinguished.  `PIDS[*]` will spit out the array as one space-delimited string, and then in context it'll be interpreted as an array due to word splitting. But if one of the elements has a space, it'll be wrongly interpreted as two elements.


Here is the recommended way to loop through an array in Bash:
```suggestion
for pid in "${PIDS[@]}"; do
```
The quotes protect against word splitting, and the `@` will give the entire array rather than the stringified representation of it.

Reply:
Hm, my Shellcheck linting plugin appears to not tell me about warnings? Good to know.

Fixed: 073f7cf